src/test/groovy/com/streamr/client/utils/OrderedMsgChainSpec.groovy

```diff
@@ -241,7 +241,7 @@ class OrderedMsgChainSpec extends Specification {
             void accept(StreamMessage streamMessage) {
                 received++
             }
-        }, gapHandler, 5000L, 5000L)
+        }, gapHandler, 5000L, 5000L, false)
```

Comment:
Would like to see a test case added to either `OrderedMsgChainSpec` or some integration test to ensure the problem doesn't reappear in the future.